app/javascript/submission_handling/shared.js

```diff
@@ -0,0 +1,24 @@
+export let allChecked = (form) => form.querySelectorAll('.checkbox:checked');
+export let submitterValue = (event) => {
+    if (event.submitter) {
+        return event.submitter.value;
+    }
+
+    return 'Not defined';
+}
+
+export const pendingHoldsForm = document.querySelector('form#pending-holds');
```

Comment:
@cdmo shall we make this more generic so can be used by checkouts/renewals as well, or keep them seperate 🤔

Reply:
`submitterValue` and `allChecked` are meant to be generic... you're referring to `pendingHoldsForm` then? 🤔

could do something like `selectForm` that takes an argument of the form `id`? Kinda what you thinking or?

Reply:
yes, exactly for pendingHoldsForm so can use it for checkouts form as well. Right, others are generic.

Reply:
👌 a commit coming

Reply:
great thanks!